application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -45,24 +44,25 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
+        int MAX_TAGS_THRESHOLD_WARNING = 200;
+        if (tagSystem.getAllIds().size() > MAX_TAGS_THRESHOLD_WARNING) {
```

Comment:
Now you call `tagSystem.getAllIds()` twice. But the call is potentially expensive (it reads a database). Call it only once and put it into a variable.